src/renderer/components/db-metadata-list.jsx

```diff
@@ -0,0 +1,100 @@
+import React, { Component, PropTypes } from 'react';
+
+
+export default class DbMetadataList extends Component {
+  static propTypes = {
+    tables: PropTypes.array.isRequired,
+    views: PropTypes.array.isRequired,
+    functions: PropTypes.array.isRequired,
+    procedures: PropTypes.array.isRequired,
+    database: PropTypes.object.isRequired,
+    shouldShow: PropTypes.bool.isRequired,
+    onSelectTable: PropTypes.func.isRequired,
+  }
+
+  constructor(props, context) {
+    super(props, context);
+    this.state = { Tables: true };
+  }
+
+  toggleListCollapse(toggledList) {
+    const state = { ...this.state };
+    if (!state[toggledList]) {
+      state[toggledList] = true;
+    } else {
+      state[toggledList] = false;
+    }
+    this.setState(state);
```

Comment:
Are you sure about this one though? Seems better to keep it this way and use native setState method than mutating this.state directly.